samples/cellular/modem_shell/sample.yaml

```
@@ -213,25 +213,15 @@ tests:
       - thingy91_nrf9160_ns
     platform_allow: thingy91_nrf9160_ns
     tags: ci_build
-  sample.cellular.modem_shell.memfault_modem_trace.nrf9160:
+  sample.cellular.modem_shell.modem_trace:
```

To make a distinction from modem_uart_trace, I'd rename this.
```suggestion
  sample.cellular.modem_shell.modem_trace_shell:
```

---

This should have been removed because the «send to memfault» functionality has been removed. I will clean it up. Thanks!